docs/install.rst

```diff
@@ -13,7 +13,7 @@ Requirements
 
 - `Numpy`_ |minimum_numpy_version| or later
 
-- `pytest`_ 3.1 or later
+- `pytest <https://docs.pytest.org/en/latest/>`_ 3.1 or later
```

Comment:
Didn't you bump to `pytest` 4 in `pytest-astropy`?

Reply:
I just removed this line since pytest is no longer a required dependency - and we already mention below that pytest-astropy is a test dependency - I'd rather just say that and not have to keep the minimum version of pytest in sync between pytest-astropy and the astropy docs.